dev-games/ogre/ogre-13.6.1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```

Comment:
Could you use the latest available EAPI here, 8?

Reply:
Sure, will do